packages/whats-new/src/whats-new-page.js

```diff
@@ -3,7 +3,8 @@
  */
 import React, { useEffect } from 'react';
 import { recordTracksEvent } from '@automattic/calypso-analytics';
-import { GuidePage } from '@wordpress/components';
+// eslint-disable-next-line wpcalypso/no-unsafe-wp-apis
```

Comment:
Sorry about this, but I didn't realize `Text` was experimental still 😢 I think we should probably not use it if that is the case. What do you think?